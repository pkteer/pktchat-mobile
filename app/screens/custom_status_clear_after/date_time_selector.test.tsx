// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import Preferences from '@mm-redux/constants/preferences';
import {renderWithRedux} from '@test/testing_library';

import DateTimeSelector from './date_time_selector';

describe('screens/date_time_selector', () => {
    const baseProps = {
        theme: Preferences.THEMES.denim,
        handleChange: jest.fn(),
    };

    it('should match snapshot', () => {
        const wrapper = renderWithRedux(
            <DateTimeSelector {...baseProps}/>,
        );

        expect(wrapper.toJSON()).toMatchSnapshot();
    });
});
