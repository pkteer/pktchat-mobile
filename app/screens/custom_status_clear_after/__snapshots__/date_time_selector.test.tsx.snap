// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screens/date_time_selector should match snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#ffffff",
      "flex": 1,
      "paddingTop": 10,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "space-evenly",
        "marginBottom": 10,
      }
    }
  >
    <View
      accessibilityRole="button"
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID="clear_after.menu_item.date_and_time.button.date"
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#1c58d9",
              },
            ]
          }
        >
          Select Date
        </Text>
      </View>
    </View>
    <View
      accessibilityRole="button"
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID="clear_after.menu_item.date_and_time.button.time"
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
              Object {
                "color": "#1c58d9",
              },
            ]
          }
        >
          Select Time
        </Text>
      </View>
    </View>
  </View>
</View>
`;
